wgengine/netstack/ssh.go

```diff
@@ -107,7 +107,13 @@ func (ns *Impl) handleSSH(s ssh.Session) {
 		return
 	}
 
-	cmd := exec.Command("/bin/bash")
+	var cmd *exec.Cmd
+	sshUser := s.User()
+	if os.Getuid() != 0 || sshUser == "root" {
+		cmd = exec.Command("/bin/bash")
+	} else {
+		cmd = exec.Command("/bin/su", "-", sshUser)
```

Comment:
fine for now, but maybe use `/usr/bin/env su`? e.g. this doesn't work on NixOS

Reply:
done